cli/createTemplate.sh

```diff
@@ -25,6 +27,7 @@ Usage: $(basename $0) -l LEVEL -u DEPLOYMENT_UNIT -c CONFIGURATION_REFERENCE -q
 
 where
 
+(o) -b FLOW                    is a flow through hamlet you want to invoke to perform your task
```

Comment:
update to reflect multiple flows

Reply:
You can provide the -b option multiple times, so each parameter is one flow